dpdata/abacus/relax.py

```
@@ -76,7 +76,7 @@
                         list(map(lambda x: float(x) * a0, loglines[i + k].split()[1:4]))
                     )
             else:
-                assert False, "Unrecongnized coordinate type, %s, line:%d" % (
+                assert False, "Unrecongnized coordinate type, %s, line:%d" % (  # noqa: UP031
```

Ruff (0.8.0)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)